test/RenPool.test.js

```diff
@@ -26,6 +26,27 @@ const RenToken = require('@renproject/sol/build/testnet/RenToken.json');
 const DarknodeRegistryLogicV1 = require('@renproject/sol/build/testnet/DarknodeRegistryLogicV1.json');
 const DarknodePayment = require('@renproject/sol/build/testnet/DarknodePayment.json');
 
+function base58ToHex(textData) {
```

Comment:
I usually define symbols, _e.g.,_ `functions` and `const` as local as possible. This is to ensure that those symbols are _only_ being used in that scope. Should we place it back to its respective `describe` section?

Reply:
Agree. But in the case of this functions and also increaseMonth and NODE_ID, those are values/functions that are being used in 3 different describe blocks. So in case we scope it to each block we'll triplicate the code

Reply:
Got it. And what about creating a `describe` section to host register/deregister darknode and refund bond? Then we could move these into that `describe` section.

Reply:
Ok, let's do that. :+1: